tests/validation/tests/v3_api/common.py

```diff
@@ -968,6 +968,26 @@ def dig_check():
     wait_for(callback=dig_check, timeout_message="Failed to resolve {0}".format(host))
 
 
+def validate_dns_record_deleted(client, dns_record, timeout=DEFAULT_TIMEOUT):
+    """
+    Checks whether ingress got deleted successfully.
+    Validates if ingress is null in for current object client.
+    @param client: Object client use to create ingress
+    @param dns_record: record object subjected to be deleted
+    @param timeout: Max time to keep checking whether ingress is deleted or not
+    """
```

Comment:
FIx the comments to indicate that this method works on dns_records

Reply:
Done